api/organisations/migrations/0044_auto_20230719_2157.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.20 on 2023-07-19 21:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('organisations', '0043_organisationsubscriptioninformationcache_allowed_projects'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='organisationsubscriptioninformationcache',
+            name='updated_at',
+        ),
```

Comment:
Hmm... we can't really remove fields as we'll end up with a schema mismatch. The issue is that, when we perform a release, the new release runs the migration on the database but the old application servers are still running until the new ones are healthy. The old servers will expect a field to exist that doesn't and we'll end up with those old servers throwing unhandled exceptions. 

We should just leave the `updated_at` field and add the 2 new ones.

Reply:
Done.